core/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosSchedulerBackend.scala

```diff
@@ -67,6 +67,8 @@ private[spark] class MesosSchedulerBackend(
 
   // The listener bus to publish executor added/removed events.
   val listenerBus = sc.listenerBus
+  
+  val executorCores = sc.conf.getDouble("spark.mesos.executor.cores", 1)
```

Comment:
This variable should be renamed `mesosExecutorCores`. In Spark, whenever we see the word "executor" we automatically think of Spark executors. Just having `executorCores` here is somewhat misleading.

Also, I know the rest of this file doesn't do this, but can you make this `private[mesos]` so it's clear that it's only read within this package?